app/uk/gov/hmrc/fhregistration/connectors/EnrolmentStoreProxyConnector.scala

```diff
@@ -16,37 +16,22 @@
 
 package uk.gov.hmrc.fhregistration.connectors
 
-/*
- * Copyright 2019 HM Revenue & Customs
```

Comment:
Hmm, why has this copyright notice gone?

Reply:
this file has two licenses, top there is 2021 one (before line 17 ), so i removed this 2019 license.. not sure how it got duplicated in the first place :)